catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transaction/UpdateActionImpl.java

```diff
@@ -29,9 +35,9 @@
   private Map<String, String> prefixToUriMappings;
```

Comment:
The original plan was to remove this since `getPrefixToUriMappings()` wasn't being used as far as I could tell. I decided to leave it in the interface anyway though since it's just a refactoring (the functionality shouldn't change) and the benefit of removing it (cleaning up a small bit of code) wasn't worth the risk of possibly breaking downstream projects (aside from the fact that they would have to convert to using the new interface and impl class).